src/NzbDrone.Core/Download/Aggregation/Aggregators/AggregateLanguages.cs

```diff
@@ -76,9 +76,19 @@ public RemoteEpisode Aggregate(RemoteEpisode remoteEpisode)
                 languages = languages.Except(languagesToRemove).ToList();
             }
 
-            if ((languages.Count == 0 || (languages.Count == 1 && languages.First() == Language.Unknown)) && releaseInfo is { IndexerId: > 0 } && releaseInfo.Title.IsNotNullOrWhiteSpace())
+            if ((languages.Count == 0 || (languages.Count == 1 && languages.First() == Language.Unknown)) &&
+                releaseInfo?.Title?.IsNotNullOrWhiteSpace() == true &&
+                (releaseInfo is { IndexerId: > 0 } || releaseInfo.Indexer?.IsNotNullOrWhiteSpace() == true))
```

Comment:
`(releaseInfo is { IndexerId: > 0 } || releaseInfo.Indexer?.IsNotNullOrWhiteSpace() == true)` these checks can be moved to inside this statement to reduce the duplication.

`indexer?.Settings is IIndexerSettings` below takes care of null checking against `indexer`